src/styles/StyledComponents.js

```diff
@@ -60,7 +60,6 @@ const MapDiv = styled.div`
   border-left: 0.125rem solid silver;
   border-right: 0.125rem solid silver;
 `;
```

Comment:
@khwanchaiwill I changed these styles up a bit in my last PR that has indeed been merged. I think you may need to re-clone the front-end repo. cC @npm-nic @mtruj013 @jlong5795